src/main/java/step4/model/Cars.java

```diff
@@ -0,0 +1,48 @@
+package step4.model;
+
+import step4.strategy.CarMovingStrategy;
+import step4.strategy.Strategy;
+
+import java.util.ArrayList;
+import java.util.Comparator;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class Cars {
+  private final List<Car> cars;
+
+  private final Strategy strategy;
+
+  public Cars(List<Car> cars) {
+    this.cars = cars;
+    this.strategy = new CarMovingStrategy();
+  }
+
+  public List<Car> getCars() {
+    return cars;
+  }
+
+  public boolean runCycle() {
+    cars.forEach(car -> car.move(strategy));
+    return true;
+  }
```

Comment:
```suggestion
  public boolean runCycle(Strategy strategy) {
    cars.forEach(car -> car.move(strategy));
    return true;
  }
```
이렇게 실행할 때 주입받아서 사용하면 어떨까요?